features/Context/ContentType.php

```diff
@@ -7,17 +7,26 @@
 use Behat\Behat\Context\Context;
 use Behat\Behat\Context\SnippetAcceptingContext;
 use Behat\MinkExtension\Context\RawMinkContext;
+use Exception;
 use eZ\Publish\API\Repository\ContentTypeService;
 use eZ\Publish\API\Repository\Exceptions\NotFoundException;
+use eZ\Publish\API\Repository\Repository;
 use PHPUnit_Framework_Assert as Assertion;
 
 class ContentType extends RawMinkContext implements Context, SnippetAcceptingContext
 {
     /** @var \eZ\Publish\API\Repository\ContentTypeService */
     private $contentTypeService;
 
-    public function __construct(ContentTypeService $contentTypeService)
+    /**
+     * Current content type within this context.
+     * @var \eZ\Publish\API\Repository\Values\ContentType\ContentType
+     */
+    private $currentContentType;
+
+    public function __construct(Repository $repository, ContentTypeService $contentTypeService)
```

Comment:
Here can be used the `ArgumentResolver` to inject the services instead of passing them through behat_suites.yml

``` php
/**
* @injectService $repository @ezpublish.api.repository
* @injectService $contentTypeService @ezpublish.api.service.content_type
*/
public function __construct(Repository $repository, ContentTypeService $contentTypeService)
```